handwritten_digit_recog/python/train.py

```diff
@@ -31,6 +31,7 @@
 
 from __future__ import print_function
 import argparse
+import os
```

Comment:
Hey Milan, this change is already merged, please remove it so that I can rebase this PR and merge it.

Reply:
I couldn't find a way to get rid of that commit, so do I just do another commit reverting these changes?